src/Console/Shell.php

```diff
@@ -508,8 +508,7 @@ public function runCommand($argv, $autoMethod = false, $extra = [])
             return $this->main(...$this->args);
         }
 
-        $this->err('No subcommand provided. Choose one of the available subcommands.', 2);
```

Comment:
This should be in sync with the new command runner, thats why this line should probably not be removed, right?

Reply:
I dunno. I just reverted the commit, so it wasn't there before 11179

Reply:
I think we should keep this as well.